docs/pages/User/GettingStarted/test.md

```diff
@@ -22,75 +21,110 @@ kubectl create namespace liqo-demo
+you can open a browser and directly check the connectivity to both IP addresses.
+Similarly, you can also use `curl`:
 ```
-curl --verbose $POD_HOST
+curl ${LOCAL_POD_IP}
+curl ${REMOTE_POD_IP}
 ```
 
 If you do not have direct connectivity, on the other hand, you can fire up a pod and run `curl` from inside:
-
 ```
-kubectl run --image=curlimages/curl tester -n default -it --rm -- curl --verbose $POD_HOST
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${LOCAL_POD_IP}
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${REMOTE_POD_IP}
 ```
 
 ## Expose the pod through a Service
 
-The above `hello-world.yaml` manifest tells Kubernetes to create also a service, which is designed to serve traffic to the previously deployed pod.
+The above `hello-world.yaml` manifest tells Kubernetes to create also a service, which is designed to serve traffic to the previously deployed pods.
 The service is a traditional [Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/) and can work with Liqo with no modifications.
 
-This can be seen by inspecting the service and its endpoints:
+Indeed, inspecting the service it is possible to observe that both `nginx` pods are correctly specified as endpoints.
+```
+kubectl describe service liqo-demo -n liqo-demo
+
+Name:              liqo-demo
+Namespace:         liqo-demo
+Type:              ClusterIP
+IP:                10.105.51.71
+Port:              web  80/TCP
+TargetPort:        web/TCP
+Endpoints:         10.244.2.214:80,172.16.97.219:80
+```
 
-```kubectl get endpoints -n liqo-demo```
 
-You should see the pod offloaded to the remote cluster listed as service endpoint.
+### Check the Service connectivity
 
-<!-- TODO: report the output of this command -->
+First, you have to retrieve the IP address of the service:
+```
+SVC_IP=$(kubectl get service liqo-demo -n liqo-demo --template={{.spec.clusterIP}})
+echo "Service IP: ${SVC_IP}"
+```
 
+If you have direct connectivity with the home cluster from your host (e.g. you are running K3s locally),
+you can open a browser and directly check the connectivity to the service IP address.
+Similarly, you can also use `curl`:
+```
+curl ${SVC_IP}
+```
 
-### Check the Service connectivity
+If you do not have direct connectivity, on the other hand, you can fire up a pod and run `curl` from inside:
+```
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${SVC_IP}
+```
 
-If you have direct connectivity with the cluster (e.g. K3s) from your host:
+You can also connect to the service through its _service name_, which exploits the Kubernetes DNS service:
 
-```bash
-SVC_IP=$(kubectl get service -n liqo-demo --template={{.spec.clusterIP}})
-echo $SVC_IP
 ```
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl http://liqo-demo.liqo-demo
+```
+
```

Comment:
Similarly to the section before, I'm missing the conclusions here.
What do we expect the user to see/to conclude once coming here?